### DISCUSSION TITLE
Korean translation

### DISCUSSION THREAD

**Discussion by Demian107** [microsoft/Web-Dev-For-Beginners#618] [Q&A] [answered]
Can you make korean translation

**Comment by jlooper** [comment, marked as answer]
There are several already done, look in the /translations folders: https://github.com/microsoft/Web-Dev-For-Beginners/blob/main/1-getting-started-lessons/1-intro-to-programming-languages/translations/README.ko.md - these are community built so if you'd like to complete the missing ones please feel free to raise PRs

**Reply by Demian107** [reply]
Nice! 😀
